samples/cpp/classification_sample_async/README.md

```diff
@@ -1,33 +1,32 @@
 # Image Classification Async C++ Sample {#openvino_inference_engine_samples_classification_sample_async_README}
 
-This sample demonstrates how to execute an inference of image classification networks like AlexNet and GoogLeNet using Asynchronous Inference Request API.
+This sample demonstrates how to do inference of image classification models using Asynchronous Inference Request API.  
+Models with only 1 input and output are supported.
 
 In addition to regular images, the sample also supports single-channel `ubyte` images as an input for LeNet model.
 
-Image Classification Async C++ sample application demonstrates how to use the following Inference Engine C++ API in applications:
+The following Inference Engine C++ API is used in the application:
 
-| Feature    | API  | Description |
-|:---     |:--- |:---
-|OpenVINO Runtime Version| `ov::get_openvino_version` | Get Openvino API version
-|Available Devices| `ov::Core::get_available_devices`| Get version information of the devices for inference
-| Asynchronous Infer | `ov::InferRequest::start_async`, `ov::InferRequest::set_callback` | Do asynchronous inference with callback
-|Custom Extension Kernels| `ov::Core::add_extension`, `ov::Core::set_config`| Load extension library and config to the device
-| Model Operations | `ov::set_batch`, `ov::get_batch` |  Managing of model, operate with its batch size. Setting batch size using input image count.
+| Feature | API | Description |
+| :--- | :--- | :--- |
+| Asynchronous Infer | `ov::InferRequest::start_async`, `ov::InferRequest::set_callback` | Do asynchronous inference with callback |
```

Comment:
We can have something like this `Manage the model, operate with its batch size. Set batch size using input image count.` in description column. Maybe we should use a semicolon instead of a comma?